rs/sns/governance/canister/canister.rs

```diff
@@ -733,6 +734,14 @@ fn add_maturity_(request: AddMaturityRequest) -> AddMaturityResponse {
     governance_mut().add_maturity(request)
 }
 
+#[cfg(feature = "test")]
+#[export_name = "canister_update advance_target_version"]
+fn advance_target_version(request: AdvanceTargetVersionRequest) {
```

Comment:
Why didn't this function end up in rs/sns/governance/canister/governance_test.did?

Reply:
Hmm, seems like it's not enforced by the candid type enforcement in C+

Reply:
No, the candid type enforecment seems to specifically reject it, not sure what's going on here